doc/Advanced.md

```diff
@@ -1137,6 +1137,34 @@ The following tasks will be merged correctly:
 ```
 The same approach can be applied to other actors in the QC framework, like Checks (`checkName`), Aggregators(`aggregatorName`), External Tasks (`taskName`) and Postprocessing Tasks (`taskName`).
 
+## Templating config files
+
+> [!IMPORTANT]  
+> Templating only works when using aliECS, i.e. in production and staging.
+
+To template a config file, modify the corresponding workflow in `ControlWorkflows`. This is needed because we won't use directly `Consul`  but instead go through `apricot` to template it. 
+
+1. Replace `consul-json` by `apricot`
+2. Replace `consul_endpoint` by `apricot_endpoint`
+3. Make sure to have single quotes around the URI
+
+Example:
+
+```
+o2-qc --config consul-json://{{ consul_endpoint }}/o2/components/qc/ANY/any/mch-qcmn-epn-full-track-matching --remote -b
+```
+becomes
+```
+o2-qc --config 'apricot://{{ apricot_endpoint }}/o2/components/qc/ANY/any/mch-qcmn-epn-full-track-matching --remote -b
```

Comment:
@Barthelemy the single quotes are not closed in this line

Reply:
good catch, thanks !